lib/str_util.cpp

```diff
@@ -465,6 +465,17 @@ string timediff_format(double diff) {
 
 }
 
+[[nodiscard]] int64_t ParseStringtoInt64(const std::string& str)
```

Comment:
We try to keep libboinc compatible with the c++03 compilers, so please remove `nodiscard` here.

Reply:
Roger.